src/main/java/ua/lviv/iot/App.java

```diff
@@ -0,0 +1,42 @@
+package ua.lviv.iot;

```

Comment:
This package name is good, but not good enough. Imaging someone will start using your app as a library (to import and reuse your models classes for example) – would it be clear what they deal with if they import ua.lviv.iot.* ? Not really, some more specifics would be better.

Reply:
Fixed